### DISCUSSION TITLE
posts/Go/template/

### DISCUSSION THREAD

**Discussion by Q1mi** [Q1mi/BlogComments#356] [Announcements]
https://www.liwenzhou.com/posts/Go/go_template/ 

 李文周的Blog Go语言 template 包 前端 模板 HTML

**Comment by huoshuaibing** [comment]
{{template "ul.tmpl"}}  是不是写错了，引用的话，正确写法是 {{ template "ul.tmpl"  . }}  详情见资料：https://learnku.com/articles/35560

**Comment by Q1mi** [comment]
> @huoshuaibing
>  {{template "ul.tmpl"}}  是不是写错了，引用的话，正确写法是 {{ template "ul.tmpl"  . }}  详情见资料：https://learnku.com/articles/35560

感谢反馈，需要给模板传值的时候就要加 `.`。
